Ryujinx/Ui/SwitchSettings.cs

```diff
@@ -0,0 +1,424 @@
+            _controller1Type     .SetActiveId(SwitchConfig.ControllerType.ToString());
+
+            _lStickUp1.Label     = SwitchConfig.KeyboardControls.LeftJoycon.StickUp.ToString();
+            _lStickDown1.Label   = SwitchConfig.KeyboardControls.LeftJoycon.StickDown.ToString();
+            _lStickLeft1.Label   = SwitchConfig.KeyboardControls.LeftJoycon.StickLeft.ToString();
+            _lStickRight1.Label  = SwitchConfig.KeyboardControls.LeftJoycon.StickRight.ToString();
+            _lStickButton1.Label = SwitchConfig.KeyboardControls.LeftJoycon.StickButton.ToString();
+            _dpadUp1.Label       = SwitchConfig.KeyboardControls.LeftJoycon.DPadUp.ToString();
+            _dpadDown1.Label     = SwitchConfig.KeyboardControls.LeftJoycon.DPadDown.ToString();
+            _dpadLeft1.Label     = SwitchConfig.KeyboardControls.LeftJoycon.DPadLeft.ToString();
+            _dpadRight1.Label    = SwitchConfig.KeyboardControls.LeftJoycon.DPadRight.ToString();
+            _minus1.Label        = SwitchConfig.KeyboardControls.LeftJoycon.ButtonMinus.ToString();
+            _l1.Label            = SwitchConfig.KeyboardControls.LeftJoycon.ButtonL.ToString();
+            _zL1.Label           = SwitchConfig.KeyboardControls.LeftJoycon.ButtonZl.ToString();
+            _rStickUp1.Label     = SwitchConfig.KeyboardControls.RightJoycon.StickUp.ToString();
+            _rStickDown1.Label   = SwitchConfig.KeyboardControls.RightJoycon.StickDown.ToString();
+            _rStickLeft1.Label   = SwitchConfig.KeyboardControls.RightJoycon.StickLeft.ToString();
+            _rStickRight1.Label  = SwitchConfig.KeyboardControls.RightJoycon.StickRight.ToString();
+            _rStickButton1.Label = SwitchConfig.KeyboardControls.RightJoycon.StickButton.ToString();
+            _a1.Label            = SwitchConfig.KeyboardControls.RightJoycon.ButtonA.ToString();
+            _b1.Label            = SwitchConfig.KeyboardControls.RightJoycon.ButtonB.ToString();
+            _x1.Label            = SwitchConfig.KeyboardControls.RightJoycon.ButtonX.ToString();
+            _y1.Label            = SwitchConfig.KeyboardControls.RightJoycon.ButtonY.ToString();
+            _plus1.Label         = SwitchConfig.KeyboardControls.RightJoycon.ButtonPlus.ToString();
+            _r1.Label            = SwitchConfig.KeyboardControls.RightJoycon.ButtonR.ToString();
+            _zR1.Label           = SwitchConfig.KeyboardControls.RightJoycon.ButtonZr.ToString();
+
+            _custThemePath.Buffer.Text           = SwitchConfig.CustomThemePath;
+            _graphicsShadersDumpPath.Buffer.Text = SwitchConfig.GraphicsShadersDumpPath;
+            _fsLogSpinAdjustment.Value           = SwitchConfig.FsGlobalAccessLogMode;
+
+            _gameDirsBox.AppendColumn("", new CellRendererText(), "text", 0);
+            _gameDirsBoxStore   = new ListStore(typeof(string));
+            _gameDirsBox.Model  = _gameDirsBoxStore;
```

Comment:
```suggestion
            _gameDirsBox.Model = _gameDirsBoxStore;
```
And update the line before too